lib/conjur/conjur_config.rb

```diff
@@ -63,5 +71,13 @@ def trusted_proxies_valid?
     rescue
       false
     end
+
+    def authenticators_valid?
```

Comment:
What do you think about checking for a valid authenticator (`authn-k8s`, `authn-ldap`, `authn-oidc`, etc.) rather than just the form of a possible authenticator?